readme.md

```diff
@@ -162,13 +195,13 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 </html>
 ```
 
-> Your answer...
+Yield should be <%= yield %>. Something should show up as long as there is another template being passed in through to yield.
 
 ## Question 13
 
 Of the three options below, which is the most "correct" way of organizing the files that make up an Angular app, as used in class? Why is this option considered "better" than the other two?
 
-> Your answer...
+B, considered better because the folders are split into the models that are being passed through to the page helping to bring clarity as to where the data is getting pulled from/passed through. Kind of freestyling this answer.  
 
```

Comment:
:+1:  B is correct. Angular is made up of templates and components that are very closely-linked. In this layout, related components are listed right next to each other in alphabetical order.

A and C would require a lot of jumping from folder to folder because they don't reflect the intertwinedness of HTML and JS in Angular. C does not reflect the "one view, one controller" paradigm.